packages/ember-glimmer/tests/integration/event-dispatcher-test.js

```diff
@@ -138,6 +139,36 @@ moduleFor('EventDispatcher#setup', class extends RenderingTest {
     this.$('div').trigger('myevent');
   }
 
+  ['@test canDispatchToEventManager is deprecated'](assert) {
+    this.dispatcher.canDispatchToEventManager = null;
+    this.registerComponent('x-foo', {
+      ComponentClass: Component.extend({
+        eventManager: {
+          myEvent() {
+            assert.ok(true, 'custom event was triggered');
```

Comment:
It's not clear from how this test is written that this line of code is actually executing. Can you set a boolean and check it at the end, or use assert.expect?